### ISSUE TITLE
Ability to interact with system fields

### ISSUE THREAD

**Issue by drupalshift** [rezo-labs/directus-extension-computed-interface#42]
Currently, it seems you can't use the computed field with date_created and date_updated
It would be nice if you could read those fields

**Comment by duydvu** [comment]
You're right. It is a bug since those fields are not updatable in the UI so the computed field won't update.

**Comment by duydvu** [comment]
Hi @drupalshift , you can upgrade to v1.8.0 and enable the "Initial Compute". It will force the computed field to be recalculated whenever you open the form.

Be careful with `date_updated` as it will be updated right after saving and trigger the compute again.
